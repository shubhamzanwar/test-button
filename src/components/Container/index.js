import React from 'react';
import Input from '../Input'
import Button from '../Button'
import * as styles from './index.module.css'

class Container extends React.Component{
    render(){
        return <div>
            <Input/>
            <Button />
        </div>

    }

}
export default Input;