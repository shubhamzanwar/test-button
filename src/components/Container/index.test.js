import React from 'react';
import Container from './index';
import { render, fireEvent } from '@testing-library/react';

describe('the Container component', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<Container />);

        expect(asFragment()).toMatchSnapshot();
    });
});