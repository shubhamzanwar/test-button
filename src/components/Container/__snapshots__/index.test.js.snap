// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Container component should render correctly 1`] = `
<DocumentFragment>
  <input
    type="text"
  />
</DocumentFragment>
`;
